### ISSUE TITLE
Bayes Ellipses overlap vs MaxLikOverlap inconsistencies

### ISSUE THREAD

**Issue by ktsievers** [AndrewLJackson/SIBER#71]
I am having some problems with the Bayesian Ellipses overlap being very different from the MaxLik Overlap. 

I have run through the demo.siber.data and the MaxLik overlap and the Bayes overlap values are essentially the same. Obviously slightly different bc the MaxLik is one calculation, and the Bayes does many resampling. But the difference is 0.285, and 0.290.

I have followed through the vignette with my data, in the exact same method and Im getting completely different values between the two overlap methods. The only change I made is using my data set for createSiberObject. My MaxLik shared overlap space is 0.70, and the mean overlap for the bayes values are 0.27. The area1, area2, and overlap values are also quite different between these two methods. 

The only difference in my data is the dbl values (for the iso1 and iso2) are much longer past the decimal (which I wouldn't think would cause an issue). And there is some sample size differences between the comparisons (15 and 11). 

So I ran the overlap calculations for a comparison (same data set) with n=21 vs n=26.  The MaxLik overlap = 0.24, and the BayesOverlap = 0.50. And again the areas are very different between the two overlap methods. I just cant work out whats going on, and why those values would be so different.

**Comment by AndrewLJackson** [comment]
can i ask what group labels you are using? I want to check that the handling of the generated factors is correct between the two.

**Comment by AndrewLJackson** [comment]
I added a basic check of the ML and Bayesian estimates for overlap here in the tmp folder of the Master branch. You would need to clone or download this repo as the tmp folder is omitted on package build() or grab the file and the dataset and move them somewhere on your local machine. I have tested with a few of the group-pairs and it seems to be behaving correctly. For your own analysis, make sure p.interval argument is set to the same value in both calls. Also make sure to set the draws argument in bayesianOverlap() to be something sensible such as draws = 1000 so you get sensible coverage of the posterior. It defaults to draws = 10 because it can be slow to run and that will not produce comparable results.
